app/utills.py

```diff
@@ -0,0 +1,26 @@
+import httpx
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
It's a good practice to provide a default value when using `os.getenv()`. This way, if the environment variable is not set, your application will not crash and will use the default value instead. For example: `API_KEY = os.getenv('API_KEY', 'default_value')`.